server/models/account.js

```diff
@@ -27,6 +27,23 @@ const schema = Joi.object({
         id: Joi.string().required(),
         name: Joi.string().lowercase().required()
     })
+}).label('Account object').example({
+  _id: {asdf: 'asdf'},
+  name: {
+      first: 'John',
+      middle: 'Doe',
+      last: 'Smith'
+  },
+  notes: [ NoteEntry.schema._examples[0] ],
+  status: {
+      current: StatusEntry.schema._examples[0],
+      log: [StatusEntry.schema._examples[0]]
+  },
+  timeCreated: '2018-05-23T20:19:27Z',
+  user: {
+      id: 'asdfsadffds',
+      name: 'username?'
+  }
```

Comment:
These schema labels/examples feel cluttered (again could just new on my eyes). I'm also concerned about the  potential for added maintenance overhead and the example drifting from the schema.

Reply:
Valid concerns. I just reused what models I saw in the routes just to reduce redundancy but we could keep the meta data type label/description/example methods out of the models and only in routes or just leave that out. Labels only show up in the "model" tab of the documentation (and name recorded for re-usable models in Swagger.json). The Joi generated meta data that I think is ultimately most useful is the endpoint descriptions and query param descriptions. Everything else is just a bonus. 

The use of example() here is actually a bad example because it's easy to drift from schema like you said. Typically you would add a .example() to every property in your response instead of one big example() on the root of the model like this. When you do that Swagger still generates the same complete example JSON to display in documentation and it shouldn't drift. It does clutter the code either way but it can be useful to reference even in the code for folks unfamiliar with it. One way to reduce the clutter is to put most of the validation on new lines if you prefer it: https://www.dropbox.com/s/gn9s0o0ih65rq6q/Screenshot%202018-05-24%2019.36.15.png?dl=0